### ISSUE TITLE
In Windows RABBITMQ_PLUGINS_DIR default value is wrong if RABBITMQ_BASE is set

### ISSUE THREAD

**Issue by ilmirajat** [rabbitmq/rabbitmq-server#433]
I tried to install RabbitMQ to windows server so that RABBITMQ_BASE was set to non-default folder (in this case "D:\ProgramFiles\RabbitMQServer-Data"). The rest of the RabbitMQ was installed to the default folder in this case "C:\Program Files (x86)\RabbitMQ Server\rabbitmq_server-3.5.6".

Attempt to install management plugin failed. Rabbitmq-service.bat did not expand correctly ez-files to the expanded-folder under Mnesia-folder.

Solution was to set RABBITMQ_PLUGINS_DIR environment variable explicitly. (In this case I also copied plugins to another folder, because instructions warned that in some case windows folders names causes problems.) 

The problem seems to be in rabbitmq-env.bat:

```
if "!RABBITMQ_PLUGINS_DIR!"=="" (
	if "!PLUGINS_DIR!"=="" (
		set RABBITMQ_PLUGINS_DIR=!RABBITMQ_BASE!\plugins
	) else (
		set RABBITMQ_PLUGINS_DIR=!PLUGINS_DIR!
	)
)
```

If RABBITMQ_PLUGINS_DIR or PLUGINS_DIR is not set, then the path would be RABBITMQ_BASE\plugins i.e. in my case D:\ProgramFiles\RabbitMQServer-Data\plugins"). However it should be "C:\Program Files (x86)\RabbitMQ Server\rabbitmq_server-3.5.6\plugins".

Probably line 3 above should be:

```set RABBITMQ_PLUGINS_DIR=!RABBITMQ_HOME!\plugins```

(Because in the beginning of the file ```set RABBITMQ_HOME=%SCRIPT_DIR%..```)

Currently, the service I needed to install works as it should and I don't have time to do pull request right now. I posted this issue and solution, just in case someone else have the same problem. If some one else have not fixed this, I'll do pull request once I had have time to test this solution in a clean environment.

**Comment by michaelklishin** [comment]
Your understanding [seems to be correct](https://github.com/rabbitmq/rabbitmq-server/blob/master/scripts/rabbitmq-defaults#L42), thank you.

**Comment by michaelklishin** [comment]
Fixed in #444.
